src/js/helpers/graph/declarationHelper.js

```diff
@@ -34,7 +34,7 @@ export const getNodeGroup = (node) => {
 
 const createVisNode = (node) => ({
   id: node.id,
-  label: node.label,
+  label: `<b>${node.label}</b>`,
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b, using `b` tag is considered as a bad practice.

Reply:
yes i agree. But in this case as a result markup we don't get <b> tag. We get only one canvas tag for all graph. I tried to do that by using span with class, but in effect i get: 

<img width="1440" alt="Screenshot 2020-05-18 at 09 53 19" src="https://user-images.githubusercontent.com/29116555/82188220-d7001780-98ed-11ea-9423-259894228f18.png">

otherwise it is consistent with vis.js documentation.